src/GitVersionCore/Helpers/KeyComparer.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+
+namespace GitVersion.Helpers
+{
+    public class LambdaKeyComparer<TSource, TKey> : Comparer<TSource> where TSource : class
+    {
+        private readonly Func<TSource, TKey> _keySelector;
```

Comment:
I think much of the rest of the codebase doesn't use `_` as a prefix for local fields, but instead `this.` to resolve.

Reply:
Agree